src/googleclouddebugger/python_util.cc

```diff
@@ -79,7 +79,7 @@ bool CodeObjectLinesEnumerator::Next() {
 
   while (true) {
     offset_ += next_entry_[0];
-    line_number_ += next_entry_[1];
+    line_number_ += (int8_t)next_entry_[1];
```

Comment:
Should probably use appropriate c++ style cast vs c style.

Reply:
Switched to static_cast.